samples/unity-of-bugs/Assets/Scripts/SceneLoaderBehaviour.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Assets.Scripts
+{
+    public sealed class SceneLoaderBehaviour : MonoBehaviour

```

Comment:
Same with sealed. 
I would like for us to keep the naming consistent. We have BugFarm so this could be BugFarmSceneLoader or even just SceneLoader.

Reply:
This class contains actions like `LoadTransitionScene` or `LoadBugFarmScene`. Why to rename it to `BugFarmSceneLoader`? It's not `BugFarm` related. `TransitionScene` calls `LoadBugFarmScene` for eaxmple.

What about `sealed`?

Reply:
I'll rename `BugFarm` to `BugFarmBehaviour` then.

I don't agree with your point on `sealed` classes. `sealed` classes usually can be further optimized by a compiler, so I'm not buying into "visual noise".